test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -44,6 +44,7 @@ var (
 	podsPerNode    = flag.Int("pods_per_node", 1, "Number of serve_hostname pods per node")
 	upTo           = flag.Int("up_to", 1, "Number of iterations or -1 for no limit")
 	maxPar         = flag.Int("max_par", 500, "Maximum number of queries in flight")
+	gke            = flag.String("gke", "", "Taget GKE cluster with this context")
```

Comment:
May want to add, either in a comment or the help text, that the pattern for gke kubeconfig context is `gke_{project}_{zone}_{cluster-name}`.